src/Landing/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ChevronDown from './ChevronDown';
 import headshot from '../img/headshot.jpg';
-import * as styles from './index.module.css';
+import styles from './index.module.css';
```

Comment:
Not using the default export syntax is probably the reason why you added `src/declaration.d.ts`. Fixed it so no need to use extra declarations.